src/main/java/frc/robot/subsystems/Climber_Subsystem.java

```diff
@@ -87,73 +165,49 @@ public double getPitch() {
         return ahrs.getPitch();
     }
 
-    public boolean getVerticalRLimit() {
-        return verticalR.getSensorCollection().isFwdLimitSwitchClosed();
-    }
-
-    public boolean getVerticalLLimit() {
-        return verticalL.getSensorCollection().isFwdLimitSwitchClosed();
-    }
-
-    public boolean getHorizontalRLimit() {
-        return horizontalR.getSensorCollection().isFwdLimitSwitchClosed();
-    }
-
-    public boolean getHorizontalLLimit() {
-        return horizontalL.getSensorCollection().isFwdLimitSwitchClosed();
-    }
-
     public int getRVerticalPos() {
```

Comment:
I didn't actually change anything functionally in these methods. Just made them output to the smart dashboard. You'll want the raw positions to make sure the soft limits are working properly.